src/Uno.UI.Toolkit/UIElementExtensions.cs

```diff
@@ -67,6 +67,10 @@ internal static void SetElevationInternal(this DependencyObject element, double
 			if (element is Android.Views.View view)
 			{
 				AndroidX.Core.View.ViewCompat.SetElevation(view, (float)Uno.UI.ViewHelper.LogicalToPhysicalPixels(elevation));
+#if __ANDROID_28__
+				view.SetOutlineAmbientShadowColor(shadowColor);
+				view.SetOutlineSpotShadowColor(shadowColor);
```

Comment:
Have you tested on a Android 7 device? I don't think the API is available, it will cause a crash :/
https://developer.android.com/reference/android/view/View#setOutlineAmbientShadowColor(int)

Reply:
#if __ANDROID_28 __ will be true all the time since we compile against a MonoAndroid90 and 10 already (28 is 9), so I think it can be removed.